main.nf

```diff
@@ -477,6 +489,12 @@ ch_cadd_wg_snvs_tbi = params.cadd_wg_snvs_tbi ? Channel.value(file(params.cadd_w
 ch_pon = params.pon ? Channel.value(file(params.pon)) : "null"
 ch_target_bed = params.target_bed ? Channel.value(file(params.target_bed)) : "null"
 
+// parameters to experiment with Control-FREEC
+cf_window = params.cf_window ? Channel.value(file(params.cf_window)) : "null"
```

Comment:
Don't think we should use `file()` when it's a string actually.
Especially when you never use `cf_window` anymore but `params.cf_window`

Reply:
```suggestion
```

Reply:
Yep, C&P , fixed, tack.